manifest-template.js

```diff
@@ -4,84 +4,84 @@ const fs = require('fs')
 const [hash, url] = process.argv.slice(2)
 
 const manifest = {
-    "app-id": "com.tutanota.Tutanota",
-    "runtime": "org.freedesktop.Platform",
-    "runtime-version": "21.08",
-    "base": "org.electronjs.Electron2.BaseApp",
-    "base-version": "21.08",
-    "branch": "stable",
-    "sdk": "org.freedesktop.Sdk",
-    "command": "tutanota-desktop",
-    "separate-locales": false,
-    "finish-args": [
-        "--share=ipc",
-        "--socket=x11",
-        "--socket=wayland",
-        "--socket=pulseaudio",
-        "--socket-system-bus",
```

Comment:
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

how did I do this

Reply:
rather ask how that's not an error that breaks builds